container/dockershell.sh

```diff
@@ -171,17 +181,23 @@ fi
 mkdir -p ${USER_VOLUME}
 # start container in interactive mode
 
-entryPoint="env && ${SETUP_PATH}/setup-ssl.sh && ${SETUP_PATH}/entrypoint_msg.sh && bash --login"
+echo "setting the entrypoint variable"
+echo $SETUP_PATH
+entryPoint="env && ${SETUP_PATH}/setup-ssl.sh && ${SETUP_PATH}/setup-container.sh ${ES_VERSION} && ${SETUP_PATH}/entrypoint_msg.sh && bash --login"
```

Comment:
Hmm, I see why you were concerned about the time it took to download the libraries, I had not understood that we would set this up at runtime as the container was started up each time. @hroumani could we discuss this ?

Reply:
I touch on this in the comments below, we'll chat.